app/views/otus/batch_load/castor/_batch_load.html.erb

```diff
@@ -0,0 +1,35 @@
+<div>
+  <h2> Castor batch load </h2>
```

Comment:
Perhaps this could be completed in a separate pull request? There would need to be multiple changes in different files and file renaming that would need to be done. I could just change the title for now so on the outlook it looks like "OTU with identifier" as well.